src/vmm/src/pipe.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright 2020 Sartura All Rights Reserved.
```

Comment:
What is the reason for having these wrappers here instead of in the `utils`?

Reply:
It seemed appropriate since it's related to the ```serial.rs``` file if You disagree we can change it?